com.unity.perception/Documentation~/GroundTruth/KeypointLabeler.md

```diff
@@ -23,8 +23,9 @@ keypoints {
 
 The `state` entry has three possible values: 
 * 0 - the keypoint either does not exist or is outside of the image's bounds
-* 1 - the keypoint exists inside of the image bounds but cannot be seen because the object is not visible at its location in the image
-* 2 - the keypoint exists and the object is visible at its location
+* 1 - the keypoint is defined and inside of the image's bounds but cannot be seen because it is either occluded by 
```

Comment:
'by another object or itself' is a bit awkward, since keypoints don't actually occlude themselves - it's the mesh that occludes the keypoint. Maybe we just simplify this to 'inside of the image bounds but is occluded'